src/Main.ts

```diff
@@ -104,12 +143,41 @@ class Main {
     public domItem(index: number): Container | undefined {
         return Loader.domItem(index);
     }
+
+    /**
+     * addCustomShape
+     * @param type
+     * @param drawer
+     */
+    public addCustomShape(type: string, drawer: IShapeDrawer | ShapeDrawerFunction): void {
+        let customDrawer: IShapeDrawer;
+        if (typeof drawer === "function") {
+            customDrawer = {
+                draw: drawer
+            };
+        } else {
+            customDrawer = drawer;
+        }
+
+        ShapeUtils.addShapeDrawer(type, customDrawer);
+    }
+
+    /**
+     * addCustomPreset
+     * @param preset
+     * @param options
+     */
+    public addCustomPreset(preset: string, options: RecursivePartial<IOptions>): void {
+        Presets.addPreset(preset, options);
+    }
 }
 
 const tsParticles = new Main();
 
 Object.freeze(tsParticles);
 
+export default tsParticles;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)